packages/agents/cartographer/poller/src/lib/operations/transfers.ts

```diff
@@ -12,23 +12,23 @@ import {
 import { getContext } from "../../shared";
 
 const getMaxNonce = (transfers: DestinationTransfer[] | XTransfer[]): number => {
-  return transfers.length == 0 ? 0 : Math.max(...transfers.map((transfer) => transfer.xparams.nonce ?? 0));
+  return transfers.length == 0 ? 0 : Math.max(...transfers.map((transfer) => transfer.origin?.xcall.txNonce ?? 0));
```

Comment:
Since xcall already has a nonce I don't think we need this, right?

Reply:
Yes, right. but I was going to process them consistently

Reply:
Right. But having nonce already makes this unnecessary, so I think it's better to just remove it.